Explorer/Assets/DCL/Landscape/TerrainGenerator.cs

```diff
@@ -87,6 +89,28 @@ public TerrainGenerator(IMemoryProfiler profilingProvider, bool measureTime = fa
 
             // TODO (Vit): we can make it an array and init after constructing the TerrainModel, because we will know the size
             terrains = new List<Terrain>();
+            terrainChunkColliders = new List<Collider>();
+        }
+
+        public void SetTerrainCollider(Vector2Int parcel, bool isEnabled)
+        {
+            int offsetX = parcel.x - terrainModel.MinParcel.x;
```

Comment:
Do you need to make this math on every change? Looks like this is already done on start, I would suggest just doing a dictionary.